internal/dataplane/configfetcher/config_fetcher.go

```diff
@@ -94,7 +94,7 @@ func (cf *DefaultKongLastGoodConfigFetcher) TryFetchingValidConfigFromGateways(
 	}
 	if goodKongState != nil {
 		if cf.fillIDs {
-			goodKongState.FillIDs(logger)
+			goodKongState.FillIDs(logger, "")
```

Comment:
Shouldn't we pass the workspace ID here?

Reply:
Updated.